packaging/resource_suite_s3_nocache.py

```diff
@@ -2393,6 +2394,31 @@ def test_iput_with_N_flag_issue_2198(self):
             # clean up the file that was put in case it wasn't cleaned up
             self.user1.assert_icommand('irm -f %s' % filename)
 
+    def recursive_put_small_files_with_checksum_and_N_flag_issue_2198(self):
+
+        dirname = 'recursive_put_small_files_with_checksum_and_N_flag_issue_2198'
```

Comment:
do we usually use "__issue"... in these names?

Reply:
Usually.  I made the change.